.github/workflows/black.yml

```diff
@@ -1,11 +1,36 @@
 name: Black Formatter
-on: [push, pull_request]
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches_ignore: []
+
 jobs:
-  black:
+  formatting:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
```

Comment:
v2 ?

Reply:
I hadn't reviewed version 2 yet (notes here) https://github.com/actions/checkout#whats-new but it seems reasonable, we might benefit from a tinier speedup of checkout! I'm happy to update this.

Reply:
Updated.